forecast.ts

```diff
@@ -1,5 +1,6 @@
-import JiraApi from 'jira-client';
+import JiraApi from "jira-client";
```

Comment:
Good idea to run Prettier over this, but can you rebase it into a separate commit to make the PR easier to review?

Reply:
Yeap, that's actually a good idea. Doing it now.

Reply:
As discussed with @kn-ad, for the sake of churning out more code today skipping this. Pulling down branch and reviewing highlighted character change would give a better view for PR review.